tests/testsuite.src/run_fundamental.at

```diff
@@ -11193,3 +11193,1003 @@ AT_CHECK([$COMPILE prog.cob], [0], [], [])
+      *>   
+           MOVE -1.0000000000000000000000000000000000001 TO FIELD-03 .
+           MOVE -9223372036854776135 TO EXPO-INT .
+           COMPUTE RESULT-MIN   = FIELD-03 ** EXPO-INT   
+                   ON SIZE ERROR
+                      DISPLAY '8.1 : Should not size error'
+                   NOT ON SIZE ERROR
+                        IF RESULT-MIN NOT = 
+                          -0.9999999999999999990776627963145223869
+                        THEN
+                            DISPLAY 
+                              '8.1: Result <' RESULT-MIN 
+                         '> != 0.9999999999999999990776627963145223869'
+           END-COMPUTE .
+      *> 
+      *> --------------------------------------------------------------
+      *> 
+           MOVE 99999999999999999999999999999999999999 TO FIELD-MAX .
+           MOVE 1.0000000000000000000000000000000000001 TO EXPO-MIN .
+           COMPUTE RESULT-HUDGE  = FIELD-MAX ** EXPO-MIN   
+                   ON SIZE ERROR
+                      CONTINUE 
+                   NOT ON SIZE ERROR
+                        DISPLAY '9: SHOULD SIZE ERROR'           
+           END-COMPUTE .
+      *> 
+      *> --------------------------------------------------------------
+      *> 
+           MOVE 35 TO FIELD-02  .
+           MOVE 26.1234 TO EXPO-DEC .
+           COMPUTE RESULT-HUDGE  = FIELD-02  ** EXPO-DEC   
+                   ON SIZE ERROR
+                      CONTINUE 
+                   NOT ON SIZE ERROR
+                        DISPLAY '10: SHOULD SIZE ERROR'           
+           END-COMPUTE .
+      *> 
+      *> --------------------------------------------------------------
+      *> 
+           MOVE 35.1234 TO FIELD-02  .
+           MOVE 0  TO EXPO-DEC .
+           COMPUTE RESULT-HUDGE  = FIELD-02  ** EXPO-DEC   
+                   ON SIZE ERROR
+                        DISPLAY '11: SHOULD NOT SIZE ERROR'           
+                   NOT ON SIZE ERROR
+                      CONTINUE 
+           END-COMPUTE .
+           IF RESULT-HUDGE NOT = 1
+           THEN
+               DISPLAY '11: RESULT <' RESULT-HUDGE '> != 1'
+           END-IF .          
+      *> 
+      *> --------------------------------------------------------------
+      *> 
+           MOVE -53.1234 TO FIELD-02  .
+           MOVE 0  TO EXPO-DEC .
+           COMPUTE RESULT-HUDGE  = FIELD-02  ** EXPO-DEC   
+                   ON SIZE ERROR
+                        DISPLAY '12: SHOULD NOT SIZE ERROR'           
+                   NOT ON SIZE ERROR
+                      CONTINUE 
+           END-COMPUTE .
+           IF RESULT-HUDGE NOT = 1
+           THEN
+               DISPLAY '12: RESULT <' RESULT-HUDGE '> != 1'
+           END-IF .          
+      *> 
+           STOP RUN .
+])
+AT_CHECK([$COMPILE prog.cob], [0], [], [])
+AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [], [])
+AT_CLEANUP
+
+AT_SETUP([Power size error and limits cases 2])
+AT_KEYWORDS([POWER])
```

Comment:
Words that are in AT_SETUP are automatically added to the keyword list. Replacing it with the main statement - here COMPUTE - would be useful (also applies above).